trie/utils.go

```diff
@@ -1,199 +0,0 @@
-// Copyright 2022 The go-ethereum Authors
```

Comment:
Why deleting this `tracer` struct? It was assumed to track the change of trie node in master branch, right?

Reply:
Yep. It's aimed to track the change of trie nodes, especially for "deleted" nodes which are not captured in committer.
But in sake of simplification and some other considerations, it's a bit over-complicated and won't be used right now.